### PR TITLE
feat(unlock-protocol-com): Add redesigned jobs, privacy, and terms page

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

This moves jobs, privacy, and terms page to our redesign. 

I've migrated terms and privacy page into a markdown file instead because that way, it is easily updateable by a lawyer (who may be non-technical).

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol-com/next.config.js [hunk 1]
```diff
@@ -46,17 +46,16 @@ Object.keys(requiredConfigVariables).forEach((configVariableName) => {
...
-module.exports = {
+
+const nextConfig = {
   publicRuntimeConfig: requiredConfigVariables,
   webpack(config) {
     config.module.rules.push({
       test: /\.md$/,
       use: 'raw-loader',
     })
-    config.module.rules.push({
```
Comment (julien51) [inline unlock-protocol-com/next.config.js:56]: Let's move to another PR? It looks like a signigicant config change that I don't want to break :)

Comment (searchableguy) [reply unlock-protocol-com/next.config.js:56]: Okay. Though it's not used anymore.

**Review by julien51 — APPROVED** [review]
I like the idea of moving to Markdown!
Can you clean up the `PrivacyContent`, `JobsContent` and `TermsContent` too? After that LG!

**Comment by searchableguy** [comment]
I will remove that all in later PR. The whole content folder can be safely removed once I've migrated everything.
